### PR TITLE
Updated Scanner Installer and Uninstall

### PR DESCRIPTION
**Pull request by yarunachalam**
- Include Pre-req for Cluster create
- Install Scanner Updater
- Uninstall Scanner Update and NeuVector Extension
- To check the Deployed NeuVector version

### CODE REVIEW

File: content/neuvector-prime/azure/contents.lr [hunk 1]
```diff
@@ -169,6 +169,19 @@ Once logged in, you can begin to [navigate and configure NeuVector]. Refer (http
 
    !["Keyvaluepair config"](./keyvaluepairconfig.png)
 
+
+## Install Scanner Updater
+
+The NeuVector scanner image is updated daily with a new CVE database on the NeuVector docker hub registry.It is recommended to install scanner updater AFTER successfull initial NeuVector deployment.
```
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: super nit-picky but add a space after the period before `It`

File: content/neuvector-prime/azure/contents.lr [hunk 2]
```diff
@@ -169,6 +169,19 @@ Once logged in, you can begin to [navigate and configure NeuVector]. Refer (http
...
+
+Execute the following command in a Cloud Shell
+
+```
+helm repo add
```
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: its really just `helm repo add` with no parameters or anything???

File: content/neuvector-prime/azure/contents.lr [hunk 3]
```diff
@@ -169,6 +169,19 @@ Once logged in, you can begin to [navigate and configure NeuVector]. Refer (http
...
+```
+
+
+
 ## Auto upgrade minor version
```
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: this whole section will no longer apply if we're installing the scanner independent of the CNAB... I'm not 100% sure thats been decided

**Review by jeremy-moffitt — CHANGES_REQUESTED** [review]
If we're going ahead with the separate scanner install, we should remove the auto-update section.

File: content/neuvector-prime/azure/contents.lr (hunk 2 above)
Comment (yarunachalam) [reply content/neuvector-prime/azure/contents.lr]: waiting for final command.. we will finalize tomorrow.

File: content/neuvector-prime/azure/contents.lr [hunk 4]
```diff
@@ -169,26 +169,40 @@ Once logged in, you can begin to [navigate and configure NeuVector]. Refer (http
...
 
-## Auto upgrade minor version
 
-By default the Kubernetes Service Extension property "Auto upgrade minor version" is enabled
-during the NeuVector offer deployment. This will allow NeuVector to regularly update
-its CVE database.
+## Install Scanner Updater
 
-Click "Refresh" in the Extension + application screen to view the latest Version.
+The NeuVector scanner image is updated daily with a new CVE database on the NeuVector docker hub registry. It is recommended to install scanner updater AFTER successfull initial NeuVector deployment.
```
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: Isn't installing the scanner updater *required* after installing the NeuVector offer?
Also s/successfull/successful/

File: content/neuvector-prime/azure/contents.lr [hunk 5]
```diff
@@ -220,6 +234,34 @@ Billing will be available in Azure Portal billing
 Home > Cost Management <subscription> | Cost analysis
 
 
+## Uninstalling NeuVector and Scanner Updater
+
+Before Uninstalling NeuVector, uninstall scanner updater.
```
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: add a "the" between "uninstall" and "scanner" .... "uninstall the scanner updater."

**Review by jeremy-moffitt — CHANGES_REQUESTED** [review]
Couple of tweaks requested... Also, is the yaml file supposed to be double spaced?

Not related to this change, but the flow of the section about the Azure portal showing the password is kinda awkward and we should look at updating it...

File: content/neuvector-prime/azure/contents.lr (hunk 3 above)
Comment (yarunachalam) [reply content/neuvector-prime/azure/contents.lr]: done

File: content/neuvector-prime/azure/contents.lr (hunk 4 above)
Comment (yarunachalam) [reply content/neuvector-prime/azure/contents.lr]: Updated the flow..

File: content/neuvector-prime/azure/contents.lr [hunk 6]
```diff
@@ -4,17 +4,17 @@ theme: suse
...
-##  How to install NeuVector and configure Billing-adapter.
+##  How to install NeuVector and configure Billing-adapter
 
 The NeuVector offer handles billing by gathering usage data from NeuVector. For complete installation, Ensure to follow all the steps
 till NeuVector Usage Record and Billing.
 
 ##Prerequisites
 
-These instructions assume you have an AKS cluster installed.
+These instructions assume you have an AKS cluster installed as per the [NeuVector System requirements document](https://open-docs.neuvector.com/basics/requirements)
```
Comment (rjschwei) [inline content/neuvector-prime/azure/contents.lr]: s/installed/running/ or s/installed/set up/

File: content/neuvector-prime/azure/contents.lr [hunk 7]
```diff
@@ -4,17 +4,17 @@ theme: suse
...
 
 
-A step by step walk-through for creating a new offer from the Marketplace Overview page.
+A step by step walk-through for creating a new offer from the Azure Marketplace Overview page.
```
Comment (rjschwei) [inline content/neuvector-prime/azure/contents.lr]: The customer does not create an "offer". I suspect was meant to explain how to complete a new deployment of NeuVector into a running cluster?

**Review by rjschwei — CHANGES_REQUESTED** [review]

File: content/neuvector-prime/azure/contents.lr (hunk 2 above)
Comment (yarunachalam) [reply content/neuvector-prime/azure/contents.lr]: this is not applicable now.

**Review by jeremy-moffitt — APPROVED** [review]
thanks for addressing my feedback and fixing the yaml file.

**Review by jeremy-moffitt — APPROVED** [review]

**Pull request merged**